resources/scss/ceres/widgets/Grid/_grid-widget.scss

```diff
@@ -109,4 +109,8 @@
         background-color: $white;
         padding: $spacer;
     }
-}
\ No newline at end of file
+}
+
+html.ie .widget-grid .widget-inner-stacked {
+    display: block;
```

Comment:
Müssen wir die Regel auf den IE beschränken oder wirkt sie sich negativ in anderen Browsern aus?

Reply:
Ist halt ein Eingriff in den Bootstrap Style